controllers/nexus/resource/validation/validation.go

```diff
@@ -80,7 +80,21 @@ func (v *Validator) SetDefaultsAndValidate(nexus *v1alpha1.Nexus) (*v1alpha1.Nex
 }
 
 func (v *Validator) validate(nexus *v1alpha1.Nexus) error {
-	return v.validateNetworking(nexus)
+	validators := []func(*v1alpha1.Nexus) error{v.validateDeployment, v.validateNetworking}
+	for _, v := range validators {
+		if err := v(nexus); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func (v *Validator) validateDeployment(nexus *v1alpha1.Nexus) error {
+	if nexus.Spec.Replicas > 1 {
+		v.log.Warn("Nexus server only supports 1 replica.", "Desired Replicas", nexus.Spec.Replicas)
```

Comment:
Also, we should avoid whitespace in log keys to keep consistent with the rest of the project.

```suggestion
		v.log.Warn("Nexus server only supports 1 replica.", "DesiredReplicas", nexus.Spec.Replicas)
```

Wdyt?

Reply:
Looks like this one didn't make it into the changes @ricardozanini. Did you forget it or do you not agree? hehe :-)

Reply:
LOL